From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Sun, 18 Sep 2022 16:55:20 +0200
Subject: test: Extend default.cnf with a CipherString

By setting OPENSSL_TLS_SECURITY_LEVEL=2 the 90-test_sslapi.t for the
test_set_tmp_dh (iteration 1) and test_dh_auto will fail.

Extend the default.cnf by adding "CipherString = DEFAULT:@SECLEVEL=1" as
default to ensure it is not limited by default security level.

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 test/default.cnf | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/test/default.cnf b/test/default.cnf
index f29d0e92bae8..6ebf19c997c7 100644
--- a/test/default.cnf
+++ b/test/default.cnf
@@ -5,9 +5,16 @@ config_diagnostics = 1
 
 [openssl_init]
 providers = provider_sect
+ssl_conf = ssl_sect
 
 [provider_sect]
 default = default_sect
 
+[ssl_sect]
+system_default = system_default_sect
+
 [default_sect]
 activate = 1
+
+[system_default_sect]
+CipherString = DEFAULT:@SECLEVEL=1
